apps/test/unit/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -21,4 +21,35 @@ describe('SingleSectionSetUp', () => {
       .simulate('change', {target: {value: 'Section 1'}});
     expect(updateSectionSpy).to.have.been.calledOnce;
   });
+
+  it('renders the grade options', () => {
+    const updateSectionSpy = sinon.spy();
+    const wrapper = mount(
+      <SingleSectionSetUp
+        sectionNum={1}
+        section={{}}
+        updateSection={updateSectionSpy}
+      />
+    );
+
+    const checkbox = wrapper.find('MultiSelectButton[value="K"]');
```

Comment:
Instead of using `mount` here, you could use `shallow` and check that the right props are passed into `MultiSelectGroup`, which I think is what you're testing here?

Reply:
i think i got this test mixed up as i was refactoring things, there's no reason for it to have a spy either. i'll see if i can change it into something useful